com.regnosys.rosetta.tests/src/com/regnosys/rosetta/generator/java/expression/ListOperationTest.xtend

```diff
@@ -96,7 +98,7 @@ class ListOperationTest {
 						
 						protected List<Foo.FooBuilder> assignOutput(List<Foo.FooBuilder> filteredFoos, List<? extends Foo> foos) {
 							filteredFoos = toBuilder(MapperC.of(foos)
-								.filterItem(_item -> (Boolean)areEqual(_item.<Boolean>map("getInclude", __foo -> __foo.getInclude()), MapperS.of(Boolean.valueOf(true)), CardinalityOperator.All).get()).getMulti());
+								.filterItem(_«defaultImplicitVariable.name» -> (Boolean)areEqual(_«defaultImplicitVariable.name».<Boolean>map("getInclude", __foo -> __foo.getInclude()), MapperS.of(Boolean.valueOf(true)), CardinalityOperator.All).get()).getMulti());
```

Comment:
I've made tests robust against changes in the name of implicit variables (e.g., `item`, `it`, `this`, ...) It's now defined in a centralised class `ImplicitVariableUtil`.